packages/nimble-components/src/rich-text-viewer/index.ts

```diff
@@ -11,7 +18,91 @@ declare global {
 /**
  * A nimble styled rich text viewer
  */
-export class RichTextViewer extends FoundationElement {}
+export class RichTextViewer extends FoundationElement {
+    /**
+     * Markdown string to render its corresponding rich text content in the component.
+     */
+    @observable
+    public markdown!: string;
+
+    public viewer!: HTMLDivElement;
```

Comment:
Our convention is to mark `public` members that aren't really meant to be accessed by clients as `@internal`:
```ts
/**
  * @internal
*/
public viewer!: HTMLDivElement;
```

Reply:
Added the comment.